kerlescan/view_helpers.py

```diff
@@ -127,3 +129,25 @@ def log_username(logger, request):
             )
         else:
             logger.debug("identity header not sent for request")
+
+
+def validate_uuids(system_ids):
+    """
+    helper method to test if a UUID is properly formatted. Will raise an
+    exception if the format is wrong.
+    """
+    malformed_ids = []
+    for system_id in system_ids:
+        # the UUID() check was missing some characters, so adding regex first
+        if !(re.match('^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$', system_id, re.ASCII)):
```

Comment:
surprisingly, the part that this is failing to parse in `black` is the exclamation point.

Can you switch it to `if not re.match...`?

Also, this regex will succeed on this very sleepy string "ZZZZZZZZ-ZZZZ-ZZZZ-ZZZZ-ZZZZZZZZZZZZ" which is not a uuid. However, it will get caught below by the UUID check. I would either switch `\w` to `[a-fA-F0-9]`, or just add a comment saying we are enforcing the spacing here and checking for hex digits below in the next check.